### PR TITLE
Adds setting to disable about link

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

**Comment by stvstnfrd** [comment]
+1

**Comment by caesar2164** [comment]
screen shot 2017-10-25 at 5 52 51 pm

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -184,10 +184,11 @@ def should_show_debug_toolbar(_):
     'language': LANGUAGE_MAP,
 }
 
-FEATURES['ENABLE_COURSE_DISCOVERY'] = True
+FEATURES['ENABLE_COURSE_DISCOVERY'] = False
```
Comment (stvstnfrd) [inline lms/envs/devstack.py]: Is this just for your testing?

Do we need these?

File: lms/envs/devstack.py [hunk 2]
```diff
@@ -184,10 +184,11 @@ def should_show_debug_toolbar(_):
...
 # Setting for overriding default filtering facets for Course discovery
 # COURSE_DISCOVERY_FILTERS = ["org", "language", "modes"]
-FEATURES['COURSES_ARE_BROWSEABLE'] = True
+FEATURES['COURSES_ARE_BROWSABLE'] = False
```
Comment (stvstnfrd) [inline lms/envs/devstack.py]: Is this just for your testing?

Do we need these?

File: lms/envs/common.py [hunk 3]
```diff
@@ -349,6 +349,8 @@
     # Show Language selector.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    'SHOW_ABOUT_LINK': False,
```
Comment (stvstnfrd) [inline lms/envs/common.py]: This needs to default to true, to maintain back-compat.

File: lms/envs/devstack.py [hunk 4]
```diff
@@ -184,10 +184,11 @@ def should_show_debug_toolbar(_):
...
+FEATURES['SHOW_ABOUT_LINK'] = False
```
Comment (stvstnfrd) [inline lms/envs/devstack.py]: Is this just for your testing?

Do we need these?

**Comment by caesar2164** [comment]
Ugh, thanks for the catch @stvstnfrd

**Comment by caesar2164** [comment]
[Fix Register Button for CME](https://trello.com/c/zN53fcnS/199-fix-register-button-for-cme)

**Comment by caesar2164** [comment]
[Confirm that SYSADMIN link is present on suclass](https://trello.com/c/tVsm0zWr/202-confirm-that-sysadmin-link-is-present-on-suclass)

**Pull request merged**